HOWTOSTART.md

```diff
@@ -0,0 +1,334 @@
+        <ExtraSchemaExtensionId>00000000-0000-0000-0000-000000000000</ExtraSchemaExtensionId>
+        <TitleText>Demo</TitleText>
+        <Name>Demo</Name>
+        <DataServiceSelectTimeout>120</DataServiceSelectTimeout>
+        <AuthorizationProvider>Origam.Security.OrigamDatabaseAuthorizationProvider, Origam.Security</AuthorizationProvider>
+        <ProfileProvider>Origam.Security.OrigamProfileProvider, Origam.Security</ProfileProvider>
+        <LoadExternalWorkQueues>false</LoadExternalWorkQueues>
+        <ExternalWorkQueueCheckPeriod>180</ExternalWorkQueueCheckPeriod>
+        <TraceEnabled>true</TraceEnabled>
+        <ModelProvider>Origam.OrigamEngine.FilePersistenceBuilder, Origam.OrigamEngine</ModelProvider>
+      </OrigamSettings>
+    </ArrayOfOrigamSettings>
+  </xmlSerializerSection>
+</OrigamSettings>
+```
+Make sure you set the correct **ModelSourceControlLocation** in the file. Open the 
+Architect again. You should see the demo projects' packages. Double click the **Widgets**
+package and a **Deployment Scripts Pending** pop up should show up. Click Yes. The database 
+should now contain the generated tables and data.
+
+Now we add a test user into the database. Open **SQL Server Management Studio** go to
+the database **origam-demo** and run the following script
+```tsql
+DECLARE @userName NVARCHAR(max)
+DECLARE @passwordHash NVARCHAR(max)
+
+SET @userName = 'testUser'
+SET @passwordHash = 'FA000.AGJMo11O/0jTchE97kPtXfSzDM7qOaBltse7bOAINgMYNdyf7iv4P0DINxkdTdxRhA==' -- hash of: 9ECZJ83w4UNQPjrR
+
+INSERT INTO [dbo].[BusinessPartner] (
+	UserName
+	,Name
+	,Id
+	)
+VALUES (
+	@userName
+	,@userName
+	,'69aacf26-300e-477b-b9a6-408324ca1cad'
+	);
+
+INSERT INTO [dbo].[OrigamUser] (
+	UserName
+	,EmailConfirmed
+	,refBusinessPartnerId
+	,Password
+	,Id
+	,FailedPasswordAttemptCount
+	,Is2FAEnforced
+	)
+VALUES (
+	@userName
+	,1
+	,'69aacf26-300e-477b-b9a6-408324ca1cad'
+	, @passwordHash
+	,'f0154207-4f49-476d-b955-6f587dd61708'
+	,0
+	,0
+	);
+
+INSERT INTO [dbo].[BusinessPartnerOrigamRole] (
+	Id
+	,refBusinessPartnerId
+	,refOrigamRoleId
+	)
+VALUES (
+	'd8130a52-a52c-49bb-b80d-565fa5b9eb21'
+	,'69aacf26-300e-477b-b9a6-408324ca1cad'
+	,'E0AD1A0B-3E05-4B97-BE38-12FF63E7F2F2'
+	);
+
+UPDATE  [dbo].[OrigamParameters]
+SET [BooleanValue] = 1
+WHERE Id = 'e42f864f-5018-4967-abdc-5910439adc9a'
+
+```
+
+### Run the Server
+To run the demo project in Origam server you first have to build the client application.
+To do that open the folder `frontend-html` in your javascript IDE or CMD.exe and run the following.
+```
+npm install --global yarn
+```
+```
+yarn 
+```
+```
+yarn build
+```
+now you should have a production build of the front end application in 
+```
+frontend-html\dist
+```
+
+Next we have to create **OrigamSettings.config** for the server application.
+Go to Visual Studio set the startup project to **Origam.Server** and switch the solution configuration to **Debug Server**.
```

Comment:
... and set _Startup Projects_ field to `Origam.Server`

Reply:
line 166 does say "set the startup project to **Origam.Server**"